src/test/java/collab/SearchTest.java

```diff
@@ -21,86 +21,68 @@ public void setUp() {
-      return 1;
-    }
-    return 0;
+    if(isMatch) assertEquals(foundList.get(0).getEmployeeNumber(), answer.getEmployeeNumber());
+    if(!isMatch) assertEquals(foundList.size(),0);
   }    
 
-  public int searchTest_Cl(Employee answer) {
-    List<Employee> empList = empDB.searchItems("cl", answer.getCareerLevel());
+  public void searchTest_Cl(boolean isMatch, Employee answer) {
+    List<Employee> foundList = empDB.searchItems("cl", answer.getCareerLevel());
 
-    if(empList.size()==1) {
-      assertEquals(empList.get(0).getEmployeeNumber(), answer.getEmployeeNumber());
-      return 1;
-    }
-    return 0;
+    if(isMatch) assertEquals(foundList.get(0).getEmployeeNumber(), answer.getEmployeeNumber());
+    if(!isMatch) assertEquals(foundList.size(),0);
   }
 
-  public int searchTest_PhoneNum(Employee answer) {
-    List<Employee> empList = empDB.searchItems("phoneNum", answer.getPhoneNumber());
+  public void searchTest_PhoneNum(boolean isMatch, Employee answer) {
+    List<Employee> foundList = empDB.searchItems("phoneNum", answer.getPhoneNumber());
 
-    if(empList.size()==1) {
-      assertEquals(empList.get(0).getEmployeeNumber(), answer.getEmployeeNumber());
-      return 1;
-    }
-    return 0;
+    if(isMatch) assertEquals(foundList.get(0).getEmployeeNumber(), answer.getEmployeeNumber());
+    if(!isMatch) assertEquals(foundList.size(),0);
   }
 
-  public int searchTest_BirthDay(Employee answer) {
-    List<Employee> empList = empDB.searchItems("birthday", answer.getBirthday());
+  public void searchTest_BirthDay(boolean isMatch, Employee answer) {
+    List<Employee> foundList = empDB.searchItems("birthday", answer.getBirthday());
 
-    if(empList.size()==1) {
-      assertEquals(empList.get(0).getEmployeeNumber(), answer.getEmployeeNumber());
-      return 1;
-    }
-    return 0;
+    if(isMatch) assertEquals(foundList.get(0).getEmployeeNumber(), answer.getEmployeeNumber());
+    if(!isMatch) assertEquals(foundList.size(),0);
   }
   
-  public int searchTest_Certi(Employee answer) {
-    List<Employee> empList = empDB.searchItems("certi", answer.getCerti());
+  public void searchTest_Certi(boolean isMatch, Employee answer) {
+    List<Employee> foundList = empDB.searchItems("certi", answer.getCerti());
 
-    if(empList.size()==1) {
-      assertEquals(empList.get(0).getEmployeeNumber(), answer.getEmployeeNumber());
-      return 1;
-    }
-    return 0;
-  }
-  
-  public int searchTest_Total(Employee answer) {
-    return searchTest_EmployeeNum(answer)+
-    searchTest_Name(answer)+
-    searchTest_Cl(answer)+
-    searchTest_PhoneNum(answer)+
-    searchTest_BirthDay(answer)+
-    searchTest_Certi(answer);
-  }
-
-  @Test
-  public void searchTestFound() {
-    assertTrue(searchTest_Total(answer)==6);
-  }
-  
-  @Test
-  public void searchTestNotFound() {
-    assertTrue(searchTest_Total(noAnswer)==0);
+    if(isMatch) assertEquals(foundList.get(0).getEmployeeNumber(), answer.getEmployeeNumber());
```

Comment:
이 부분도 answer가 기대값인 것 같습니다. 두 값의 자리를 바꾸는 것은 어떨까요?

```suggestion
    if(isMatch) assertEquals(answer.getEmployeeNumber(), foundList.get(0).getEmployeeNumber());
```